client/packages.go

```diff
@@ -72,3 +72,15 @@ func (client *Client) Packages() (map[string]Package, error) {
 
 	return packages, nil
 }
+
+// Package returns details of a single Package
```

Comment:
This func returns an item of type `Package` but if you think the Godoc comment makes better sense without the capitalised type name then I'll all for it.